docker/template.conf

```diff
@@ -1,6 +1,4 @@
-include "application.conf"
-
-logger.scala.slick = INFO
+include required(classpath("application.conf"))
```

Comment:
Here too, is the `.conf` correct?

Note: I use the include without it, but it may well work both ways, and I didn't try it.

Reply:
Well, I'm using what was there before actually :D so that's not mine, it was here before and I don't want to do to much changes in that regard, which will change actually nothing.